utils/exporters/blender/addons/io_three/exporter/api/object.py

```diff
@@ -374,7 +378,7 @@ def matrix(obj, options):
         parent_inverted = obj.parent.matrix_world.inverted(mathutils.Matrix())
         return parent_inverted * obj.matrix_world
     else:
-        return AXIS_CONVERSION * obj.matrix_world
```

Comment:
> What was the thought process behind that method?

Unfortunately, the guy that wrote that code is no longer active in the project...